templates/saas/model1/src/app/root/components/Navbar.tsx

```diff
@@ -3,20 +3,27 @@ import NavItem from "@/components/atoms/NavItem";
 import ThemeToggler from "@/components/atoms/ThemeToggler";
 import Typography from "@/components/atoms/Typography";
 import { useLocation } from "rasengan";
+import { useState } from "react";
+import { Cancel01Icon, Menu11Icon } from "hugeicons-react";
 
 export default function Navbar() {
 	const { pathname } = useLocation();
 
 	const isActive = (path: string) => pathname === path;
+	const [displayMobileNav, setDisplayMobileNav] = useState<boolean>(false)
```

Comment:
Put that line up, just below the line 10.

I prefer to have state on the top and actions or handlers after